src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
@shadowhand I see no reason not to make the interface here. The interface can be useful to simplify Responder. 

To simplify [the Responder](https://scrutinizer-ci.com/g/equip/framework/code-structure/master/class/Equip%5CResponder%5CFormattedResponder) probably should make a separate class for "Content Negotiation" `NegotiationResponder`. Then `FormatterResponder` only be used to format a formatter (not Payload object, it has already been processed), where exactly will need such an interface for type hinting.

Also please don't rush with release and a feature freeze.